Source/Controllers/DataImport/SPDataImport.m

```diff
@@ -1453,7 +1453,7 @@ - (NSString *)mappedUpdateSetStatementStringForRowArray:(NSArray *)csvRowArray
 					[whereString appendString:@" IS NULL"];
 				} else {
 					[whereString appendString:@"="];
-					[whereString appendString:[mySQLConnection escapeAndQuoteString:cellData]];
+					[whereString appendStringOrNil:[mySQLConnection escapeAndQuoteString:cellData]];
```

Comment:
it will not make sense with the `=` appended - check before appending?

the same applies for all?

Reply:
yes you're right. let me fix